react/components/NumericStepper/index.js

```diff
@@ -136,56 +136,79 @@ export default class NumericStepper extends React.Component {
 
   render() {
     const { value, displayValue } = this.state
-    const { maxValue, minValue, size, block, label } = this.props
+    const { maxValue, minValue, size, block, label, lean } = this.props
 
     const isMin = value <= normalizeMin(minValue)
     const isMax = value >= normalizeMax(maxValue)
 
-    const buttonSizeClasses = {
-      regular: 'pv3 f6',
-      large: 'pv4 f5',
-      'x-large': 'pv5 f4',
-    }
+    // TODO: Make this less cumbersome - will be done when x-large is killed
+    const buttonSizeClasses = (lean
```

Comment:
nit: when we have nested ternary operators it may be a good idea to refactor into a function with more explicit logic.
I'd leave this one as it is, tough.

Reply:
@athoscouto agree 100%, but this whole bit is going to be refactored soon, as `x-large` sizes are going to be removed, so I went with the easy way out for now